src/sites/image/pimpandhost.com.js

```diff
@@ -1,17 +1,20 @@
 _.register({
   rule: {
     host: /^(www\.)?pimpandhost\.com$/,
-    path: /^\/image\//,
+    path: /^\/image\/\d+\?size=original/,
```

Comment:
I am at complete loss as to why this path rule fails to match the url `https://pimpandhost.com/image/24753379?size=original` !!

Reply:
original path rule, does work for that site.